pkg/web/web.go

```diff
@@ -61,56 +60,10 @@ func New(conf config.Config, store store.Store, network network.Network) (s *Ser
 		IdleTimeout:       120 * time.Second,
 	}
 
-	return s, nil
-}
-
-// If the API is disabled, this middleware factory function returns a middleware that
-// returns 529 Unavailable on all API requests. If the API is enabled, it returns nil
-// and should not be used as a middleware function.
-func (s *Server) APIEnabled() gin.HandlerFunc {
-	if s.conf.Web.APIEnabled {
-		return nil
-	}
-
-	return func(c *gin.Context) {
-		// If this is an API request and the API is disabled, return unavailable.
-		if IsAPIRequest(c) {
-			c.AbortWithStatus(http.StatusServiceUnavailable)
-			return
-		}
-
-		// Otherwise serve the UI request.
-		c.Next()
-	}
-}
+	// Update the scene with the configuration
+	scene.WithConf(&conf)
```

Comment:
This is where the scene is getting updated with the configuration; we'll have to remember where this is … I'm sure we're going to tear our hair out looking for it eventually.